packages/otel-better-auth/build.ts

```
@@ -0,0 +1,70 @@
+import { build } from "esbuild";
+
+const MINIFY = true;
+const SOURCEMAP = true;
+
+const MAX_SIZE = 50_000; // 50KB max for instrumentation package
+
+type ExternalPluginFactory = (external: string[]) => Plugin;
+const externalCjsToEsmPlugin: ExternalPluginFactory = (external) => ({
+  name: "external",
+  setup(builder): void {
+    const escape = (text: string): string =>
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)